components/console/helpers/cursor.rst

```diff
@@ -0,0 +1,99 @@
+.. index::
+    single: Console Helpers; Cursor Helper
+
+Cursor Helper
+=============
+
+.. versionadded:: 5.1
+
+    The :class:`Symfony\\Component\\Console\\Cursor`
+    class was introduced in Symfony 5.1.
+
+The :class:`Symfony\\Component\\Console\\Cursor` allows you to change the
+cursor position in a console command. This allows you to write on any position
+of the output:
+
+.. image:: /_images/components/console/cursor.gif
+   :align: center
+
+
+    // src/Commande/MyCommand.php
+    use Symfony\Component\Console\Command\Command;
+    use Symfony\Component\Console\Cursor;
+    use Symfony\Component\Console\Input\InputInterface;
+    use Symfony\Component\Console\Output\OutputInterface;
+
+    class MyCommand extends Command
+    {
+        // ...
+
+        public function execute(InputInterface $input, OutputInterface $output)
+        {
+            // ...
+
+            $cursor = new Cursor($output);
+
+            // moves the cursor to a specific column and row position
+            $cursor->moveToPosition(7, 11);
+
+            // and write text on this position using the output
+            $output->write('My text');
+
+            // ...
+        }
+    }
+
+Using the cursor
+----------------
+
+Moving the cursor
+................
+
+There are fews methods to control moving the command cursor::
+
+    // moves the cursor 1 line up from its current position
+    $cursor->moveUp();
+
+    // moves the cursor 3 lines up from its current position
+    $cursor->moveUp(3);
+
+    // same for down
+    $cursor->moveDown();
+
+    // moves the cursor 1 column right from its current position
+    $cursor->moveRight();
+
+    // moves the cursor 3 columns right from its current position
+    $cursor->moveRight(3);
+
+    // same for left
+    $cursor->moveLeft();
+
+    // move the cursor to a specific position from its current position
+    $cursor->moveToPosition(7, 11);
```

Comment:
I haven't used this feature, but I think we have to clearify this a bit. This means 7 lines down, and 11 columns to the right? And I can use negative numbers to achieve the opposite?

Or does this expect a coordinate?

Reply:
@wouterj it means 7 cols right, 11 rows down from console init cursor
the cursor always moves position from this 0:0 position, thus need an x:y from the init yes